### PR TITLE
Fix Undo with added property field for additional lights

### PR DESCRIPTION
**Pull request by martint-unity**
---
### Purpose of this PR
Fixing this fogbuz case
https://fogbugz.unity3d.com/f/cases/1300367/
Undo is missing on Additional Lights property

### CODE REVIEW

**Review by simon-engelbrecht-soerensen — APPROVED** [review]
Looks good!

File: com.unity.render-pipelines.universal/CHANGELOG.md [hunk 1]
```diff
@@ -75,6 +75,7 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 - Fixed an issue where MSAA did not work in Editor Game View on Windows with Vulkan.
 - Fixed issue where selecting and deselecting Forward Renderer asset would leak memory [case 1290628](https://issuetracker.unity3d.com/issues/urp-scriptablerendererfeatureeditor-memory-leak-while-interacting-with-forward-renderer-in-the-project-window)
 - Fixed the default background color for previews to use the original color.
+- Fixed an undo issues for additiona light property on the UniversalRenderPipeline Asset.
```
Comment (ellioman) [inline com.unity.render-pipelines.universal/CHANGELOG.md:78]: typo `additiona `

**Review by ellioman — APPROVED** [review]

File: com.unity.render-pipelines.universal/CHANGELOG.md (hunk 1 above)
Comment (martint-unity) [reply com.unity.render-pipelines.universal/CHANGELOG.md]: fixed

File: com.unity.render-pipelines.universal/CHANGELOG.md [hunk 2]
```diff
@@ -15,6 +15,7 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 - Fixed an issue where the Cinemachine Pixel Perfect Extension might cause the Orthographic Size of the Camera to jump to 1 when the Scene is loaded. [case 1249076](https://issuetracker.unity3d.com/issues/cinemachine-pixel-perfect-camera-extension-causes-the-orthogonal-size-to-jump-to-1-when-the-scene-is-loaded)
 - Fixed an issue where render scale was breaking SSAO in scene view. [case 1296710](https://issuetracker.unity3d.com/issues/ssao-effect-floating-in-the-air-in-scene-view-when-2-objects-with-shadergraph-materials-are-on-top-of-each-other)
 - Fixed GC allocations from XR occlusion mesh when using multipass.
+- Fixed an undo issues for additional light property on the UniversalRenderPipeline Asset.
```
Comment (phi-lira) [inline com.unity.render-pipelines.universal/CHANGELOG.md:18]:
```suggestion
- Fixed undo issues for the additional light property on the UniversalRenderPipeline Asset.
```

**Review by phi-lira — APPROVED** [review]
Minor changelog update.

**Pull request merged**